src/pipedream/core/oauth_token_provider.py

```
@@ -32,3 +40,34 @@ def _refresh(self) -> str:
 
     def _get_expires_at(self, *, expires_in_seconds: int, buffer_in_minutes: int):
```

Ruff (0.14.8)

Missing return type annotation for private function `_get_expires_at`

(ANN202)